aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -899,7 +899,7 @@ Tensor& s_addmm_out_sparse_dense_cpu(
   Tensor indices = sparse_._indices();
   Tensor values      = sparse_._values();
 
-  AT_DISPATCH_ALL_TYPES_AND_COMPLEX(
+  AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND(kBFloat16,
```

Comment:
I don't think we should be enabling bfloat16 like this, in regular bfloat16 matrix multiplies intermediate accumulation is done in fp32, doing intermediate accumulation in bfloat16 (as it seems is done here) is very inaccurate.

Reply:
I see.. thanks!